streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/KTableTest.scala

```diff
@@ -29,6 +29,69 @@ import org.scalatest.{FlatSpec, Matchers}
 @RunWith(classOf[JUnitRunner])
 class KTableTest extends FlatSpec with Matchers with TestDriver {
 
+  "filter a KTable" should "filter records satisfying the predicate" in {
+    val builder = new StreamsBuilder()
+    val sourceTopic = "source"
+    val sinkTopic = "sink"
+
+    val table = builder.stream[String, String](sourceTopic).groupBy((key, _) => key).count()
+    table.filter((_, value) => value > 1).toStream.to(sinkTopic)
+
+    val testDriver = createTestDriver(builder)
+
+    {
+      testDriver.pipeRecord(sourceTopic, ("1", "value1"))
+      val record = testDriver.readRecord[String, Long](sinkTopic)
+      record.key shouldBe "1"
```

Comment:
What should the value be? I think it's `null`, right? Can we put that in the test?

Otherwise, it seems like it might actually be spitting out `1`, and we wouldn't know.

Reply:
It used to be there but it returns `null` for a `Long` and scalatest doesn't seem to work with that. I tried to `.asInstanceOf[Any]` or `null.asInstanceOf[Long]` but it's doesn't seems to work. Any idea?

Reply:
Even weirder, the `null.asInstanceOf[Long]` works on my machine but not on CI...

Reply:
Or something higher up the chain, like `assert(record.value === null)`

Reply:
Trying out this:
```
record.value shouldBe (null: java.lang.Long)
```